middle_end/flambda2/from_lambda/closure_conversion_aux.mli

```diff
@@ -74,8 +74,11 @@ module IR : sig
 
   type switch =
     { numconsts : int;
-      consts : (int * Continuation.t * trap_action option * simple list) list;
-      failaction : (Continuation.t * trap_action option * simple list) option
+      consts :
+        (int * Continuation.t * Debuginfo.t * trap_action option * simple list)
+        list;
+      failaction :
+        (Continuation.t * Debuginfo.t * trap_action option * simple list) option
```

Comment:
at one point these tuples will deserve dedicated record types, maybe a CR is enough for now

Reply:
I've added a CR.